syntax = "proto3";

message MxAds1259Store {
  uint32 reading = 1;
}
